_data/engine-cli/docker_ps.yaml

```diff
@@ -348,7 +348,6 @@ examples: |-
   ```
 
   The following filter matches all containers that have exposed TCP port in the range of `8000-8080`:
-
```

Comment:
I think this is a bug. :( I will manually change it for now but we need to put the spaces back in the `17.03.x` branch of `docker/docker`....